05-FITS/io_fits_ascii_overview.ipynb

```diff
@@ -207,12 +211,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import time\n",
+    "\n",
     "with fits.open('j94f05bgq_flt.fits.gz', mode='update') as f:\n",
-    "    f[1].header[\"history\"] = \"New history card.\"\n",
-    "    f[1].header[\"comment\"] = \"This is a 47 Tuc observation with HST\"\n",
-    "    f[1].header[\"comment\"] = \"I'd like to observe it with JWST.\"\n",
-    "    f[1].header['comment']\n",
-    "    print(f[1].header['comment'])"
+    "    f[extnum].header[\"history\"] = \"{} New history card.\".format(time.ctime())\n",
```

Comment:
I feel that we should tell people about `header.add_comment()` and `header.add_history()` and that these should be presented as best practices.